src/views/Home/props/organizerForm.js

```diff
@@ -0,0 +1,9 @@
+import { ButtonTypes } from "../../../components/Button/index.view";
```

Comment:
Same here. We should reference everything from `src`. This will become 
```suggestion
import { ButtonTypes } from "./components/Button/index.view";
```